mail_activity_board/models/mail_activity.py

```diff
@@ -1,31 +1,27 @@
+# Copyright 2016 David Juaneda - <djuaneda@sdi.es>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-# SDI
-# author: djuaneda@sdi.es
 from odoo import api, models, fields
-from odoo.tools.safe_eval import safe_eval
 
 
 class MailActivity(models.Model):
     _inherit = "mail.activity"
 
     res_model_id_name = fields.Char(
-        default="", related='res_model_id.name',
-        string="Origin", store=False, readonly=True)
+        related='res_model_id.name', string="Origin",
+        store=False, default="", readonly=True)
     duration = fields.Float(
-        related='calendar_event_id.duration',
-        string="Duration", store=False, readonly=True)
-    calendar_event_id_start = fields.Datetime(
-        default=False, related='calendar_event_id.start',
+        related='calendar_event_id.duration', string="Duration",
         store=False, readonly=True)
```

Comment:
Most likely, with `related='calendar_event_id.duration', readonly=True` it should be the same, because all other attributes are inherited from the related field itself.

Reply:
I have tried to leave the field with only "related" and "readonly" attributes and it has behaved as you have indicated, so I have changed it.